pkg/webhook/policy.go

```diff
@@ -240,36 +205,28 @@ func (h *validationHandler) getDenyMessages(res []*rtypes.Result, req admission.
 		if r.EnforcementAction == "deny" || r.EnforcementAction == "dryrun" {
 			if *logDenies {
 				log.WithValues(
-					logging.Process, "admission",
-					logging.EventType, "violation",
-					logging.ConstraintName, r.Constraint.GetName(),
-					logging.ConstraintGroup, r.Constraint.GroupVersionKind().Group,
-					logging.ConstraintAPIVersion, r.Constraint.GroupVersionKind().Version,
-					logging.ConstraintKind, r.Constraint.GetKind(),
-					logging.ConstraintAction, r.EnforcementAction,
-					logging.ResourceGroup, req.AdmissionRequest.Kind.Group,
-					logging.ResourceAPIVersion, req.AdmissionRequest.Kind.Version,
-					logging.ResourceKind, req.AdmissionRequest.Kind.Kind,
-					logging.ResourceNamespace, req.AdmissionRequest.Namespace,
-					logging.ResourceName, resourceName,
+					"process", "admission",
```

Comment:
we should use the `logging.*` names to help ensure conformance with semantic logging:

https://docs.google.com/document/d/1ap7AKOupNcR_42s8mkSh5FV9eteXTd4VCqelKst73VY/edit

Reply:
Updated.
I assume the doc should also be udpated to include the new "RequestUsername      = request_username"